inc/class-wp-forms-api.php

```diff
@@ -925,6 +927,29 @@ static function process_element( $element, &$values, &$input ) {
 	}
 
 	/**
+	 * Get WP editor templates for dynamically-inserted editors in #multiple fields
+	 *
+	 * @action wp_ajax_wp_form_get_mce_templates
+	 */
+	function ajax_get_mce_templates() {
+		$results = array();
+		$bodies = filter_input( INPUT_POST, 'content', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+
+		foreach ( $bodies as $id => $body ) {
+			ob_start();
+			$lines = preg_split( '/$/m', $body );
```

Comment:
If you are only concerned with the count, would it be better to use `substr_count()` or even `preg_match_all()`?